paddleserver/dummy_pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[tool.poetry]
+name = "workaround-for-editable-install"
+version = "0.0.1"
+description = ""
+authors = ["none"]
+
+[tool.poetry.dependencies]
+python = ">=3.8,<3.11"
```

Comment:
Should we add a small comment here that the upper version of python should be +1 minor version of the one we install in the overlay ?

Reply:
These come from the kserve/paddle packages we're installing from upstream (see [here](https://github.com/kserve/kserve/blob/v0.11.2/python/paddleserver/pyproject.toml#L13)).  Would a comment here help clarify that?  

For reasons that aren't clear to me, upstream's packages have different ranges (this is <3.12, paddle is <3.11, etc) so I've matched their ranges here.